content/en/docs/Writing policies/mutate.md

```diff
@@ -351,6 +351,190 @@ The anchor processing behavior for mutate conditions is as follows:
+          name: secret-1
+          namespace: "{{ request.object.metadata.namespace }}"
+        patchStrategicMerge:
+          metadata:
+            labels:
+              foo: bar
+```
+
+By default, the above policy will not be applied when it is installed. This behavior can be configured via `mutateExistingOnPolicyUpdate` attribute. If you set `mutateExistingOnPolicyUpdate` to `true`, Kyverno will mutate the existing secret on policy CREATION and UPDATE.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: "mutate-existing-secret"
+spec:
+  mutateExistingOnPolicyUpdate: true
+  rules:
+    - name: "mutate-secret-on-configmap-event"
+      match:
+        any:
+        - resources:
+            kinds:
+            - ConfigMap
+            names:
+            - dictionary-1
+            namespaces:
+            - staging
+...
+```
+
+### Variables Referencing Target Resources
+
+To reference data in target resources, you can define the variable `target` followed by the path to the desired attribute. For example, using `target.metadata.labels.env` references the label `env` in the target resource.
+
+This policy copies the configmaps' value `target.data.key` to their label with the key `env`.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: sync-cms
+spec:
+  mutateExistingOnPolicyUpdate: false
+  rules:
+  - name: concat-cm
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+          names:
+          - cmone
+          namespaces:
+          - foo
+    mutate:
+      targets:
+        - apiVersion: v1
+          kind: ConfigMap
+          name: cmtwo
+          namespace: bar
+        - apiVersion: v1
+          kind: ConfigMap
+          name: cmthree
+          namespace: bar
+      patchesJson6902: |-
+        - op: add
+          path: "/metadata/labels/env"
+          value: "{{ target.data.key }}"  
+```
+
+`@` is added to reference the in-line value of the target resource.
+
+This policy adds the value of `keyone` from the trigger configmap `foo/cmone` as the prefix to target configmaps in their data with `keynew`.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: sync-cms
+spec:
+  mutateExistingOnPolicyUpdate: false
+  rules:
+  - name: concat-cm
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+          names:
+          - cmone
+          namespaces:
+          - foo
+    mutate:
+      targets:
+        - apiVersion: v1
+          kind: ConfigMap
+          name: cmtwo
+          namespace: bar
+        - apiVersion: v1
+          kind: ConfigMap
+          name: cmthree
+          namespace: bar
+      patchStrategicMerge:
+        data:
+          keynew: "{{request.object.data.keyone}}-{{@}}"
+```
+
+Once a mutate existing policy is applied successfully, there will be an event and an annotation added to the target resource:
+
+```
+✗ kubectl describe deploy foobar
```

Comment:
Suggest replacing your personal command prompt marker (here and elsewhere) with the standard `$` so it's not confusing.